### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by ionutcalara**

### CODE REVIEW

File: woocommerce-gateway-paylike/includes/class-wc-gateway-paylike.php [hunk 1]
```diff
@@ -0,0 +1,557 @@
...
+                        totalTax: '<?php echo $order->get_total_tax()?>',
+                        totalShipping: '<?php echo $order->get_total_shipping()?>',
+                        customerIP: '<?php echo $this->get_client_ip() ?>',
+                    },
+                }, function (err, res) {
+                    if (err)
+                        return console.warn(err);
+
+                    var trxid = res.transaction.id;
+                    jQuery("#complete_order").append('<input type="hidden" name="transaction_id" value="' + trxid + '" /> ');
```
Comment (tjconcept) [inline woocommerce-gateway-paylike/includes/class-wc-gateway-paylike.php]: Is it OK to assume that jQuery is available in a Wordpress installation?

Comment (ionutcalara) [inline woocommerce-gateway-paylike/includes/class-wc-gateway-paylike.php]: This is done on the checkout page where jQuery is used by woocommerce so yes, its safe but we can include it just to make this look more robust.

Comment (tjconcept) [inline woocommerce-gateway-paylike/includes/class-wc-gateway-paylike.php]: Perfect. Don't do anything just for the looks 🙂

**Comment by tjconcept** [comment]
Ready to merge?

**Comment by ionutcalara** [comment]
Yes, i think this is ready for merge.

**Pull request merged**